db/migrate/20220310235942_create_admins.rb

```diff
@@ -0,0 +1,14 @@
+class CreateAdmins < ActiveRecord::Migration[7.0]
+  def change
+    create_table :admins do |t|
+      t.string :name
+      t.string :email
+      t.string :position
+      t.integer :private_number
+      t.string :password
```

Comment:
Since if you implement a session management gem such as [Devise](https://github.com/heartcombo/devise) or [Clearance](https://github.com/thoughtbot/clearance), the gem will automatically generate a user table with the email field, which would generate inconsistency with the email data that you have already generated in your tables.